common/src/main/resources/tachyon-default.properties

```diff
@@ -13,6 +13,7 @@
 
 # Default configuration properties for Tachyon
 # All the properties except Build properties are override-able via tachyon-site.properties file.
+# Setting a property to -1 denotes that this property is not set but will use system default value.
```

Comment:
This is tentative. Currently, all those properties happen to be int. 
One alternative way is to leave the property value empty in properties file, what do you think?